projects/core/src/i18n/i18next/i18next-init.ts

```diff
@@ -23,30 +26,30 @@ export function i18nextInit(
         tap((config) => {
           let i18nextConfig: InitOptions = {
             ns: [], // don't preload any namespaces
-            fallbackLng: config.i18n.fallbackLang,
-            debug: config.i18n.debug,
+            fallbackLng: config.i18n?.fallbackLang,
+            debug: config.i18n?.debug,
             interpolation: {
               escapeValue: false,
             },
           };
 
-          if (config.i18n.backend) {
-            i18next.use(i18nextHttpBackend);
+          if (config.i18n?.backend?.loadPath) {
+            i18next = i18next.use(i18nextHttpBackend);
```

Comment:
Why do we need this assignment?

Reply:
Tbh, I didn't check how it behaves without it, but I wanted to align it with their docs: https://github.com/i18next/i18next-http-backend#getting-started

Reply:
The assignment seems to be not needed.
`i18next.use()` returns `i18next`, just for chaining purposes (ergonomics of the API).

Reference: source code of `i18next.use()` https://github.com/i18next/i18next/blob/cb9bd97e26734b4f0089fd69a110149e769a54e0/src/i18next.js#L265-L298